ts/screens/profile/ProfileMainScreen.tsx

```diff
@@ -76,11 +85,30 @@ class ProfileMainScreen extends React.PureComponent<Props> {
     Toast.show({ text: "The cache has been cleared." });
   };
 
+  private logout = (keepUserData: logoutMethod) => {
```

Comment:
why keepUserData?

Reply:
Right, I forgot tu update it as well. Changed.